deploy/static/provider/aws/deploy.yaml

```diff
@@ -90,21 +90,6 @@ rules:
   - get
   - list
   - watch
-- apiGroups:
```

Comment:
@tao12345666333 was this removed purposely?

Reply:
yes! we don't need the configmap for lock